packages/v4/patternfly-docs/content/get-started/release-notes.md

```diff
@@ -0,0 +1,2790 @@
+
+### React
+
+- Removed the `boundary`, `tippyProps`, and `isAppLauncher` properties [(#8231)](https://github.com/patternfly/patternfly-react/pull/8231)
+
+- Tooltips without a `triggerRef` will now have a wrapping `<div>`, which may cause issues. Snapshots may need to be updated or, to avoid the wrapping `<div>`, add a `triggerRef` with an attached ref to the trigger element. This rule will raise a warning when a `Tooltip` is imported without a `triggerRef` or `reference` prop, but will not make any code changes [(#8733)](https://github.com/patternfly/patternfly-react/pull/8733)
+
+    **Previous implementation**
+
+    ```jsx
+    <Tooltip boundary={} tippyProps={} isAppLauncher />
+    ```
+
+    **New implementation**
+
+    ```jsx
+    <Tooltip     />
+    ```
+
+-  When using the `Tooltip` component inside of a `react-charts` component, the `Tooltip` should be wrapped inside a `foreignObject`. Otherwise, it  may not render properly due to it outputting a `<div>` element inside an `<svg>` element. This rule will raise a warning when `Tooltip` is imported from `@patternfly/react-core` and at least one other import is from `@patternfly/react-charts`, but will not update any code [(#8592)](https://github.com/patternfly/patternfly-react/pull/8592)
+
+## Wizard 
+
+### Core 
+
+- Removed style that was breaking drawer example ([#5322](https://github.com/patternfly/patternfly/pull/5322))
+
+- Reordered footer buttons ([#5148](https://github.com/patternfly/patternfly/pull/5148))
+
+### React
+
+- Deprecated the v4 implementation of `Wizard`. In order to continue using the deprecated implementation, update the import path to our deprecated package and ensure that  specifiers are aliased. We suggest updating to our newer implementation of `Wizard`. The following imports will be affected by this deprecation:
+
+    - `Wizard`
+    - `WizardContext`
+    - `WizardNav`
+    - `WizardNavItem`
+    - `WizardHeader`
+    - `WizardBody`
+    - `WizardFooter`
+    - `WizardToggle`
+[(#8821)](https://github.com/patternfly/patternfly-react/pull/8821)
+
+    **Previous implementation**
+
+    ```jsx
+    import { Wizard, Button } from '@patternfly/react-core';
+    ```
+
+    **New implementation**
+
+    ```jsx
+    import { Button } from '@patternfly/react-core';
+    import { Wizard } from '@patternfly/react-core/deprecated';
+    ```
+
+- Updated  the order of the "next" and "back" buttons so that the "next" button comes after the "back" button. This update has also been made in the Next implementation of the `WizardFooter`. We recommend updating any tests that may rely on relative selectors and any composable implementations to match this new button order. This rule will raise a warning when `Wizard` is imported from `@patternfly/react-core` or `WizardFooter` is imported from `@patternfly/react-core/next`, but it will not make any code changes[(#8409)](https://github.com/patternfly/patternfly-react/pull/8409)
```

Comment:
Should be "new" instead of "next" in the following  "This update has also been made in the Next implementation..."